modules/appeals_api/app/controllers/appeals_api/higher_level_reviews/v0/higher_level_reviews_controller.rb

```diff
@@ -3,27 +3,42 @@
 require 'appeals_api/form_schemas'
 
 module AppealsApi::HigherLevelReviews::V0
-  class HigherLevelReviewsController < AppealsApi::V2::DecisionReviews::HigherLevelReviewsController
+  class HigherLevelReviewsController < AppealsApi::ApplicationController
+    include AppealsApi::CharacterUtilities
+    include AppealsApi::IcnParameterValidation
+    include AppealsApi::JsonFormatValidation
+    include AppealsApi::MPIVeteran
     include AppealsApi::OpenidAuth
     include AppealsApi::PdfDownloads
+    include AppealsApi::Schemas
+    include AppealsApi::StatusSimulation
 
-    skip_before_action :validate_icn_header
-    skip_before_action :new_higher_level_review
-    skip_before_action :find_higher_level_review
-    skip_before_action :validate_json_format
-
-    before_action :validate_icn_parameter, only: %i[download]
-    prepend_before_action :validate_json_body, if: -> { request.post? }
+    skip_before_action :authenticate
```

Comment:
Keep in mind that this skipped `authenticate` action is from the base appeals API controller - it skips api key authentication, not oauth